.changeset/light-dragons-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Remove `@octokit/rest` from dependencies.
```

Comment:
Aaaand explain how to update existing apps :p which file and so on

Reply:
Fixed. Aaaand I also removed another unused dependency 😅